packages/devtools_shared/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   path: ^1.8.0
   shelf: ^1.1.0
   usage: ^4.0.0
-  vm_service: ">=10.1.0 <12.0.0"
+  vm_service: ">=10.1.2 <12.0.0"
```

Comment:
can we write ^10.1.2 or do we need the upper bound constraint?

Reply:
@bkonyi - the upperbound constraint is because 11.0.0 is going to be released soon, and it won't actually be a breaking change (if I understand correctly) so we want to allow both